packages/components/src/input-control/utils.ts

```diff
@@ -35,13 +35,17 @@ export function getDragCursor( dragDirection ) {
  *
  * @return {string} The CSS cursor value.
  */
-export function useDragCursor( isDragging, dragDirection ) {
+export function useDragCursor(
+	isDragging: boolean,
+	dragDirection: string
+): string {
 	const dragCursor = getDragCursor( dragDirection );
 
 	useEffect( () => {
 		if ( isDragging ) {
 			document.documentElement.style.cursor = dragCursor;
 		} else {
+			// @ts-expect-error
```

Comment:
I know it's not in the scope of this PR, but this `null` assignment to `cursor` is weird